docs/en/setup/backend/backend-k8s-monitoring.md

```diff
@@ -13,6 +13,9 @@ ways to monitor deployments on Kubernetes.
 2. Rover is a SkyWalking native eBPF agent to collect network Access Logs to support topology-aware and metrics 
    analysis. Meanwhile, due to the power of eBPF, it could profile running services written by C++, Rust, Golang, etc. 
    Read [Rover setup guide](./backend-k8s-monitoring-rover.md) for more details.
+3. If Cilium is installed in Kubernetes, use Cilium Fetcher can collect network traffic data from services. 
```

Comment:
Besides this, we should have a menu item for Cilium integration.

```suggestion
3. If Cilium is installed in Kubernetes, use Cilium Fetcher to collect network traffic data of services through Cilium Hubble APIs. 
```

Is this correct?

Reply:
Correct. And I have update the menu.